### PR TITLE
build(bazel): generalize fix for AMD name for flat module out file

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: packages/bazel/src/ngc-wrapped/index.ts [hunk 1]
```diff
@@ -214,10 +214,12 @@ export function compile({allowNonHermeticReads, allDepsCompiledWithBazel = true,
     if (fileName ===
         path.join(compilerOpts.baseUrl, bazelOpts.package, compilerOpts.flatModuleOutFile + '.ts'))
       return true;
-    // Also handle the case when angular is built from source as an external repository
+    // Also handle the case the target is in an external repository.
+    // Pull the workspace name from the target which is formatted as `@wksp//package:target`.
+    const targetWorkspace = bazelOpts.target.split('/')[0].replace(/^@/, '');
```
Comment (alexeagle) [inline packages/bazel/src/ngc-wrapped/index.ts]: the workspace is available directly
https://github.com/bazelbuild/rules_typescript/blob/master/internal/tsc_wrapped/tsconfig.ts#L28

Comment (gregmagolan) [reply packages/bazel/src/ngc-wrapped/index.ts]: That comes from here:
```
    bazel_options = {
        "workspaceName": ctx.workspace_name,
        "target": str(ctx.label),
        "package": ctx.label.package,
```
and ends up with the value `angular_bazel_example` downstream in angular-bazel-example

Comment (gregmagolan) [reply packages/bazel/src/ngc-wrapped/index.ts]: the workspace of the target is only found in `bazelOpts.target`

Comment (alexeagle) [reply packages/bazel/src/ngc-wrapped/index.ts]: ah, I see, was thinking that might be the case.
How do you know the target has a repository on the front? What if it is `//third_party:vendored_angular` ?

Comment (gregmagolan) [reply packages/bazel/src/ngc-wrapped/index.ts]: Right. In that case the targetWorkspace will end up as ''. Added that to the condition. Thanks.

**Review by alexeagle — APPROVED** [review]

**Pull request closed without merging**